From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Tue, 5 Sep 2017 13:43:18 +0900
Subject: Drop code-climate-reporter

Signed-off-by: Youhei SASAKI <uwabami@gfd-dennou.org>
---
 rack-test.gemspec | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/rack-test.gemspec b/rack-test.gemspec
index 7106d6e..de4b310 100644
--- a/rack-test.gemspec
+++ b/rack-test.gemspec
@@ -23,7 +23,6 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_development_dependency(%q<codeclimate-test-reporter>, ["~> 0.6"])
       s.add_runtime_dependency(%q<rack>, ["< 3", ">= 1.0"])
       s.add_development_dependency(%q<rake>, ["~> 12.0"])
       s.add_development_dependency(%q<rdoc>, ["~> 5.1"])
@@ -32,7 +31,6 @@ Gem::Specification.new do |s|
       s.add_development_dependency(%q<sinatra>, ["< 3", ">= 1.0"])
       s.add_development_dependency(%q<thor>, ["~> 0.19"])
     else
-      s.add_dependency(%q<codeclimate-test-reporter>, ["~> 0.6"])
       s.add_dependency(%q<rack>, ["< 3", ">= 1.0"])
       s.add_dependency(%q<rake>, ["~> 12.0"])
       s.add_dependency(%q<rdoc>, ["~> 5.1"])
@@ -42,7 +40,6 @@ Gem::Specification.new do |s|
       s.add_dependency(%q<thor>, ["~> 0.19"])
     end
   else
-    s.add_dependency(%q<codeclimate-test-reporter>, ["~> 0.6"])
     s.add_dependency(%q<rack>, ["< 3", ">= 1.0"])
     s.add_dependency(%q<rake>, ["~> 12.0"])
     s.add_dependency(%q<rdoc>, ["~> 5.1"])
